docs/en_US/Builtin_Tuner.md

```diff
@@ -323,3 +324,52 @@ tuner:
   classArgs:
     optimize_mode: maximize
 ```
+
+<br>
+
+<a name="BOHB"></a>
+
+![](https://placehold.it/15/1589F0/000000?text=+) `BOHB Advisor`
+
+> Builtin Tuner Name: **BOHB**
+
+**Installation**
+
+BOHB advisor requires [ConfigSpace](https://github.com/automl/ConfigSpace), user could use `pip3 install ConfigSpace==0.4.7` to install it.
```

Comment:
please support the installation of BOHB in `nnictl package install`

Reply:
Done.